cmd/adapter/main.go

```diff
@@ -56,9 +56,7 @@ type Adapter struct {
 }
 
 // https://github.com/kedacore/keda/issues/5732
-//
-//nolint:staticcheck // SA1019: klogr.New is deprecated.
-var logger = klogr.New().WithName("keda_metrics_adapter")
+var setupLog = ctrl.Log.WithName("keda_metrics_adapter")
```

Comment:
I understand that **klogr.New** is deprecated, but I was wondering if there's a specific reason for introducing setupLog instead of reusing **logger** variable?

Would it be possible to update the code to consistently use logger without introducing setupLog?

Reply:
With this commit, we've addressed the deprecation warning and have also aligned the logging approach with that of the KEDA operator. We can further refine the logging strategy in the next steps, and if necessary, I will submit a new pull request for additional changes.